src/main/webapp/app/tasks/search-field/search-field.component.scss

```diff
@@ -0,0 +1,16 @@
+.mat-form-field {
+  width: 100%;
+  margin-top: 30px;
+
+  * {
+    color: #101d30;
+  }
+
+  ::ng-deep &.mat-focused .mat-form-field-ripple {
```

Comment:
Thanks for noting that out. I believe the ideal approach here is use create a custom Angular Material theme instead of using some DOM-piercing selectors or changing the view encapsulation settings or even using global style. Updates are available on the next commit.